filament/CMakeLists.txt

```diff
@@ -198,7 +200,9 @@ if (FILAMENT_SUPPORTS_VULKAN)
             src/driver/vulkan/VulkanSamplerCache.cpp
             src/driver/vulkan/VulkanStagePool.cpp
     )
-    if (LINUX)
+    if (USE_EXTERNAL_GLES3)
```

Comment:
This section is for Vulkan, this test does not make sense here.

Reply:
will remove